hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/security/S3GetSecretRequest.java

```diff
@@ -167,7 +168,19 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
                 assignS3SecretValue = null;
               }
             } else {
-              // Found in S3SecretTable.
+              final OMMultiTenantManager multiTenantManager =
+                  ozoneManager.getMultiTenantManager();
+              if (multiTenantManager == null ||
+                  multiTenantManager.getTenantForAccessID(accessId) == null ||
```

Comment:
No need to check for `multiTenantManager.getTenantForAccessID(accessId) == null`.

Reply:
I have removed it now, Thank you for the review!